tabcmd/parsers/login_parser.py

```diff
@@ -21,7 +21,8 @@ def login_parser():
         subparsers = parser.add_subparsers()
         login_parser = subparsers.add_parser('login', parents=[parser])
         args = login_parser.parse_args(sys.argv[LoginParser.USER_ARG_IDX:])
-
+        print(args)
+        print("PROMPTS: ", args.prompt)
```

Comment:
These seem like debug printlines? better to use the logger so they don't get accidentally left in :)